### ISSUE TITLE
Protocol Buffers Edition 2023 Language Specification: Mismatched quotes in `edition` section

### ISSUE THREAD

**Issue by jeeger** [protocolbuffers/protocolbuffers.github.io#192]
I think I found an error in the language specification grammar, where the quotes for the [edition](https://protobuf.dev/reference/protobuf/edition-2023-spec/#edition) statement are mismatched.
From reading the specification, it seems like the edition should be specified as
```
edition = '2024"
```
which looks incorrect. The example use double quotes, and I guess the intention was to allow both double or single quotes.
```
edition = "edition" "=" (("'"decimalLit "'") | ('"' decimalLit '"')) ";"
```
(for clarification, that's double-single-double quotes and single-double-single quotes).

**Comment by Logofile** [comment]
You're right. I've submitted a change for review. Thanks for catching this!

**Comment by Logofile** [comment]
The updated is published. Thanks for pointing out the issue!
